config/unleash.php

```diff
@@ -23,7 +23,7 @@
     |
     */
 
-    'url' => env('UNLEASH_URL'),
+    'url' => env('UNLEASH_URL', 'http://localhost'),
```

Comment:
This is supposed to be a required variable, so for the time being, I'd prefer not to set a default value here.